feat(identity): add limitations table (#5865)
Kong/developer.konghq.com: app/kong-identity/auth-servers.md

{% endtable %}
<!--vale on-->

## Limitations

The following limits apply to {{site.identity}} authorization servers:

<!--vale off-->
{% table %}
columns:
  - title: Limit
    key: limit
  - title: Value
    key: value
rows:
  - limit: Requests per second (TPS)

>     value: "100 transactions per second (TPS) per {{site.konnect_short_name}} region."

Author: ```suggestion
    value: "100 transactions per second (TPS) per authorization server."
```

>     value: "100 transactions per second (TPS) per {{site.konnect_short_name}} region."

Author: ```suggestion
    value: "100 transactions per second (TPS) per authorization server."
```

  - limit: Auth servers
    value: "50 auth servers per {{site.konnect_short_name}} region."
  - limit: Clients per auth server
    value: "5,000 clients per auth server."
  - limit: Access tokens

>     value: "No limit for {{site.konnect_short_name}} Enterprise. Konnect Plus is limited to 10 million tokens per month."

Reviewer: ```suggestion
    value: "Unlimited for {{site.konnect_short_name}} Enterprise. {{site.konnect_short_name}} Plus is limited to 10 million tokens per month."
```
